README.md

```diff
@@ -10,115 +10,62 @@ This repo is meant as a template which you can use to create a Discord bot very
-3. Create your `configs.ts` file in the main folder.
-
-```ts
-// Step 1: If you do NOT use Docker, remove the `.example` from this file name so it is called `configs.ts` or copy the contents of the `configs.example.ts` file into a new `configs.ts` file. Docker will require this file.
-// Step 2: Add all your bot's information below. The only required one is token and prefix. NOTE: As long as `.gitignore` file is ignoring configs.ts your configurations will be kept private!
-// Step 3: Remove these comments if you like.
-
-export const configs = {
-  // Your bot token goes here
-  token: "",
-  // The default prefix for your bot. Don't worry guilds can change this later.
-  prefix: "!",
-  // This isn't required but you can add bot list api keys here.
-  botListTokens: {
-    DISCORD_BOT_ORG: "",
-    BOTS_ON_DISCORD: "",
-    DISCORD_BOT_LIST: "",
-    BOTS_FOR_DISCORD: "",
-    DISCORD_BOATS: "",
-    DISCORD_BOTS_GG: "",
-    DISCORD_BOTS_GROUP: "",
-  },
-  // This is the server id for your bot's main server where users can get help/support
-  supportServerID: "",
-  // These are channel ids that will enable some functionality
-  channelIDs: {
-    // When a translation is missing this is the channel you will be alerted in.
-    missingTranslation: "",
-    // When an error occurs, we will try and log it to this channel
-    errorChannelID: "",
-  },
-  // These are the role ids that will enable some functionality.
-  roleIDs: {
-    // If you have a patreon set up you can add the patreon vip role id here.
-    patreonVIPRoleID: "",
-  },
-  // These are the user ids that will enable some functionality.
-  userIDs: {
-    // You can delete the `as string[]` when you add atleast 1 id in them.
-    // The user ids for the support team
-    botSupporters: [] as string[],
-    // The user ids for the other devs on your team
-    botDevs: [] as string[],
-    // The user ids who have complete 100% access to your bot
-    botOwners: [] as string[],
-  },
-};
-```
-
-4. Start the bot `deno run -A --quiet mod.ts`
-
-**Note:** To run the bot with [PM2](https://github.com/Unitech/pm2): `pm2 start mod.ts --interpreter="deno" --interpreter-args="run -A --quiet -r" `
-
-## Step By Step with Docker
-
-You can also run this image with Docker.
+## Step By Step with Docker Compose
 
-1. Install `docker` [Getting started with docker](https://docs.docker.com/get-started/)
-2. Clone the repository
-3. Create your `configs.ts` 
-4. Build the container, from the directory containing the repository `docker build -t mybot .`
-5. Run the container with `docker run -v $(pwd)/configs.ts:/bot/configs.ts -t mybot`
 
+1. Clone the repository
+2. Get your discord bot's token (not the application id or key) from your [Discord Developer console](https://discord.com/developers/applications).
+3. Create an `.env` file in your project's root directory and add your bot's token with the command `echo TOKEN=123123123 > .env`
```

Comment:
Make this two points. 
3. Make a `.env` file
4. Paste in the following:
```

```
5. Add your token and other configurations here.